itests/hive-unit/pom.xml

```diff
@@ -212,6 +212,11 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-exporter-otlp</artifactId>
+      <version>${otel.version}</version>
+    </dependency>
```

Comment:
Why are we adding in itest? We are only having changes in the prod code

Reply:
This is required in case of users running OTEL Service for HS2 which is running locally built from source code.

Reply:
We don't package itests as part of the distribution, this is integration test module. HS2 should run locally without this itests, it ain't there in the class path either